docs/import/import_data.js

```diff
@@ -12,22 +11,24 @@ export {countDamageAndBuildings, run};
 /**
  * TODO: This should be moved to a user-readable doc and expanded.
  * Processes all user-entered data to construct the "score" asset in EarthEngine
- * and write map bounds for the disaster to Firestore. Mandatory data
- * (state-level data needs to be present for every affected state):
- * 1. National Census block shapes, normally provided by EarthEngine.
- * 2. State-level Census SNAP data, at the block group level.
- * 3. State-level SVI data, at the Census tract level.
- * 4. State-level income data, at the block group level.
- * 5. State-level FeatureCollection of all buildings.
+ * and write map bounds for the disaster to Firestore. Mandatory state-level
+ * data (needs to be present for every affected state):
+ * 1. Census block group shapes (TIGER shapefile).
+ * 2. Census SNAP data, at the block group level.
+ * 3. SVI data, at the Census tract level.
```

Comment:
nit - using both state-level and tract level here - standardize hyphenation and maybe phrase one differently?

Reply:
I agree this grammar rule is confusing! "state-level" is hyphenated because it's an adjective, modifying "data". In "Census tract level", "level" is the noun, and "Census tract" is a compound adjective, modifying it. I could hyphenate it like "Census-tract level", but the rule is that if the compound adjective is common enough, or if the individual components aren't applicable adjectives themselves, you don't need to hyphenate it. I would say that applies to both "Census tract level" and "block group level". 

However, if the wording is hard to comprehend, happy to rephrase. Can you say if something in particular is confusing?